tools/install_deps/clang-format.sh

```diff
@@ -16,4 +16,5 @@
 
 
 wget -O /usr/local/bin/clang-format-9 https://github.com/DoozyX/clang-format-lint-action/raw/master/clang-format/clang-format9
-chmod +x /usr/local/bin/clang-format-9
\ No newline at end of file
+chmod +x /usr/local/bin/clang-format-9
+ln -s /usr/local/bin/clang-format-9 /usr/local/bin/clang-format
```

Comment:
As you want. Finally I don't think that vscode intellisense is using the executable.